pkg/crc/preflight/preflight_windows.go

```diff
@@ -68,3 +68,7 @@ func SetupHost() {
 func RegisterSettings() {
 	doRegisterSettings(getPreflightChecks())
 }
+
+func CleanUpHost() {
+	doCleanUpPreflightChecks(getPreflightChecks())
```

Comment:
Should this be erroring out until it's implemented on Windows ?

Reply:
@cfergeau Sure, I can add if that make sense but we need to add it soon after initial stuff merges.

Reply:
I would say it does not hurt to have it until it's implemented, this way we'll get an error if this slips. This should be a one-liner to add such an error, right?